backend/timed/tracking/serializers.py

```diff
@@ -108,8 +108,16 @@ def validate(self, data):
         from_time = data.get("from_time", instance and instance.from_time)
         to_time = data.get("to_time", instance and instance.to_time)
 
-        # validate that to is not before from
-        if to_time is not None and to_time < from_time:
+        if to_time is None:
+            return data
```

Comment:
Should we not reject this entirely? an attendance always has start and end, so this should never happen (or, if it happens, something's wrong)

Reply:
totally, not sure how to got there in the first place